# Bentacur_Lab_Notes

1. Gene depth coverage/CNV
    -1.1 for each species extract single-copy buscos from your assemblies and make into fasta
    -1.2 add genes of interest from NCBI for that speices to the corresponding fasta
    -1.3 for each species map WGS data to fasta containing single copy buscos and focal genes
    -1.4 calculate mean coverage across all single copy buscos and then coverage across each gene of interest
    
2. Repeat characterisation of each species  
    -2.1 make repeat library from own genome  
    -2.2 get repeat library from database(s)  
    -2.3 combine  
    -2.4 get repeat content of each genome  

3. Kmer distributions for each species

Example end-result figure: 
  
  
<img
  src="/images/examplefig.png"
  style="display: inline-block; margin: 0 auto; max-width: 300px">


